Cargo.toml

```diff
@@ -57,31 +57,33 @@ features = ["display_link"]
 [target.'cfg(target_os = "windows")'.dependencies]
 parking_lot = "0.11"
```

Comment:
Of course, `windows-sys` updates quite often, so we may end up with duplicated dependencies anyhow, but this can hopefully be resolved in patch versions of `parking_lot`.

Reply:
Will try to update to windows-sys 0.32. https://github.com/Amanieu/parking_lot/pull/316 also updates parking_lot to 0.32